.github/workflows/version_tests.yaml

```diff
@@ -59,12 +59,11 @@ jobs:
     strategy:
       matrix:
         k8s_version:
-          - "1.21"
-          - "1.22"
-          - "1.23"
+          - "1.31"
         serverVersion:
-          - 3.11.14
```

Comment:
Here we stop running the kuttl multi-version test agains 3.11.
At the same time, we still have 3.11 featured in the fixtures for e2e tests, a whole lot. I'd not be surprised if it's actually all of them.
Shouldn't we bump the fixtures too (perhaps not in this PR) ?

Reply:
We should remove all 3.11 tests, but I think that could happen in another PR.